development.md

```diff
@@ -0,0 +1,21 @@
+# Development with docker
+
+To buld the image run:
+
+```bash
+docker build . -t "h5rds"
+```
+
+After that it could be runned in itterative mode mounting source code `R` directory to support reactive development without rebuild the docker container:
```

Comment:
After that it could be runned in itterative mode mounting source code `R` directory to support reactive development without rebuild the docker container: --> 

After that it could be run in iterative mode mounting source code `R` directory to support reactive development without rebuilding the docker container:

Reply:
fixed